python-package/lightgbm/dask.py

```diff
@@ -434,25 +453,49 @@ def _predict(
 
 class _DaskLGBMModel:
 
+    # self._client is set in the constructor of classes that use this mixin
+    _client: Optional[Client] = None
```

Comment:
Looks like `_client` was replaced with `_get_dask_client()` function in the latest commit and is not needed anymore.

Reply:
oh good point! Remove in https://github.com/microsoft/LightGBM/pull/3883/commits/876bfe5f6c756e946594e3316fec716ae358de18